npm/gen-root.ts

```diff
@@ -1,81 +1,82 @@
-import * as fs from "fs/promises"
```

Comment:
do you have any idea why is github showing that the whole file was changed although there are not much changes? have you changed something in formatting?